app/Main.hs

```diff
@@ -62,6 +64,18 @@ hsOutP =
     optional $ strOption $
     long "hs-out" <> metavar "DIR" <> help "Generate Haskell bindings to specified dir"
 
+elmVersionP :: Parser (Maybe ElmVersion)
+elmVersionP =
+    mkElmVersion <$>
+    (optional $ strOption $
```

Comment:
Lil' confused what eitherReader fixes. I tried to make the help message better initially, but didn't want to derive `Enum` for `ElmVersion` just to print out the possibilities.